AzureServiceBusMessageTool/Main/Application/Persistence/MySettings.cs

```diff
@@ -23,6 +19,6 @@ public class MainWindowSettings
     private const int DefaultTextBoxFontSize = AppDefaults.DefaultTextBoxFontSize;
     public bool ShouldScrollToEndOnLogContentChange { get; set; }
     public bool ShouldWordWrapLogContent { get; set; }
-
     public int LogTextBoxFontSize { get; set; } = DefaultTextBoxFontSize;
-}
\ No newline at end of file
+    public int SelectedLeftPanelTab { get; set; }
```

Comment:
this property should be called SelectedLeftPanelTabIndex

Reply:
Renamed